dali/imgcodec/decoders/opencv_fallback_test.cc

```diff
@@ -29,126 +31,120 @@ namespace imgcodec {
 namespace test {
 
 namespace {
+
```

Comment:
The tests in this file were reinventing the wheel (they were written before the reusable pattern). Changed to match the rest of the tests (using TIFFs instead of BMPs due to lack of npy reference files for BMP, it doesn't really matter which format we use)